### PR TITLE
Use eth2-clients / slashing-protection-interchange-tests & add support for v5 slashing protection interchange

### LINKED ISSUE #2001: Use eth2-clients / slashing-protection-interchange-tests
Use this release https://github.com/eth2-clients/slashing-protection-interchange-tests/releases/tag/v5.0.0
instead of our NPM package. For consistency with the main eth2 spec tests

CC: https://github.com/eth2-clients/slashing-protection-interchange-tests/issues/5#event-4253435278

### PR DESCRIPTION
**Pull request by 3xtr4t3rr3str14l**
resolves #2001 and adds support for v5 interchange (as spec'd here: https://eips.ethereum.org/EIPS/eip-3076)

### CODE REVIEW

File: packages/validator/test/e2e/slashing-protection-interchange-tests/tests/tests/generated/duplicate_pubkey_not_slashable.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (dapplion) [inline packages/validator/test/e2e/slashing-protection-interchange-tests/tests/tests/generated/duplicate_pubkey_not_slashable.json]: I don't think you want to commit this files to the source code, but download them on demand when testing

**Review by dapplion — CHANGES_REQUESTED** [review]

File: packages/validator/test/e2e/slashing-protection-interchange-tests/tests/tests/generated/duplicate_pubkey_not_slashable.json (hunk 1 above)
Comment (3xtr4t3rr3str14l) [reply packages/validator/test/e2e/slashing-protection-interchange-tests/tests/tests/generated/duplicate_pubkey_not_slashable.json]: so should we maybe delete them when the tests are done running?  or just add them to the .gitignore?

**Review by wemeetagain — DISMISSED** [review]
lgtm

**Comment by wemeetagain** [comment]
@dapplion can you re-review or dismiss your review?

File: packages/validator/test/e2e/slashing-protection-interchange-tests/run.test.ts [hunk 2]
```diff
@@ -12,92 +12,98 @@ import {
   InvalidAttestationError,
   InvalidBlockError,
 } from "../../../src/slashingProtection";
-import {downloadTests, loadTestCases as newLoadTestCases} from "./downloadTest";
+import {downloadTests, loadTestCases} from "./index.test";
```
Comment (dapplion) [inline packages/validator/test/e2e/slashing-protection-interchange-tests/run.test.ts]: Importing from .test files in not very standard. Consider refactoring

Comment (3xtr4t3rr3str14l) [reply packages/validator/test/e2e/slashing-protection-interchange-tests/run.test.ts]: it didn't really need to be named a test file, so i just renamed it.

**Review by dapplion — APPROVED** [review]

**Pull request merged**